### PR TITLE
Moving all system test scripts in system_tests/.

### PR DESCRIPTION
**Pull request by dhermes**
Also removing `system_tests/__init__.py` so it is no longer a package and making all imports happen locally (rather than from the root of the project).

Changes originally inspired by emulator script breakages in #1373.

----

@jonparrott Having to worry about which scripts can see `system_tests` is somewhat of an argument for moving into `gcloud.system_tests` (discussion in #1451). Happy to go that route here instead. WDYT?

As-is, the current fix makes `pylint` angry, so at the very least we should discuss what to do about **that**.

### CODE REVIEW

**Comment by jonparrott** [comment]
Can we still exclude our system tests if they're a sub-package? If not, can we at least make the package private?

**Comment by dhermes** [comment]
Sure. Excluding them will mean they don't show up in an `sdist`, but this probably doesn't matter to `tox` (I think the main library gets `pip installed`-ed by `tox` with `-e` set, which is editable/developer mode)

**Comment by jonparrott** [comment]
SGTM. Let's try that.

**Comment by dhermes** [comment]
@jonparrott Had a revelation that I could just move those scripts into the `system_tests` directory. DERP.

**Comment by jonparrott** [comment]
I can't believe I didn't realize that either. LGTM.

File: MANIFEST.in [hunk 1]
```diff
@@ -1,4 +1,3 @@
 include README.rst
 graft gcloud
 global-exclude *.pyc
-recursive-exclude system_tests *
```
Comment (dhermes) [inline MANIFEST.in]: For posterity, this is no longer necessary since `system_tests/__init__.py` is gone

**Pull request merged**